src/components/Feedback.vue

```
@@ -73,14 +73,37 @@
+import { getCredential } from '@/utils/auth'
+import aws4 from 'aws4'
 
 const name = ref('')
 const email = ref('')
 const message = ref('')
 
 const submitFeedback = async () => {
+
+  const path = '/email'
+  const method = 'POST'
+  const session = await getSession()
```

ESLint

[error] 'getSession' is not defined.

(no-undef)